aether-kernel/aether/kernel/api/tests/test_models.py

```diff
@@ -391,3 +391,18 @@ def test_model_deletion(self):
         self.assertTrue(models.Schema.objects.filter(pk=schema.pk).exists())
         schema.refresh_from_db()
         self.assertIsNone(schema.family)
+
+    def test_schema_decorator_topic_name(self):
+        project = models.Project.objects.create(
+            name='project',
+        )
+        schema = models.Schema.objects.create(
+            name='schema',
+            definition=EXAMPLE_SCHEMA,
+        )
+        schemadecorator = models.SchemaDecorator.objects.create(
+            name='schema decorator',
+            project=project,
+            schema=schema,
+        )
+        self.assertEqual(schemadecorator.topic, {'name': 'schema decorator'})
```

Comment:
Could you add a couple of silly test in which you change the schemadecorator name and the topic doesn't change and the changing the topic one?